### PR TITLE
chore: Remove merge id column message after CREATE LABEL

### PR DESCRIPTION
**Pull request by gtyun**
Add function to distinguish label from table.
And fix to do not print message with label.

### CODE REVIEW

**Comment by protodef** [comment]
What about these messages?
`NOTICE:  merging multiple inherited definitions of column "id"`
`NOTICE:  merging multiple inherited definitions of column "properties"`

**Comment by gtyun** [comment]
Remove messages in multiple inheritance too.
Thank you for review.

File: src/backend/commands/graphcmds.c [hunk 1]
```diff
@@ -290,3 +292,37 @@ CheckDropLabel(ObjectType removeType, Oid labid)
...
+	nsptuple = SearchSysCache1(NAMESPACEOID, ObjectIdGetDatum(nspid));
+
+	if (!HeapTupleIsValid(nsptuple))
+		elog(ERROR, "cache lookup failed for label (OID=%u)", nspid);
+
+	nspdata = (Form_pg_namespace) GETSTRUCT(nsptuple);
+
+	ReleaseSysCache(nsptuple);
+
+	nspname = NameStr(nspdata->nspname);
```
Comment (protodef) [inline src/backend/commands/graphcmds.c]: Use of `nsptuple` after releasing it looks wrong.

Comment (gtyun) [inline src/backend/commands/graphcmds.c]: Fixed it. Thank you.

**Comment by protodef** [comment]
I've rebased and merged it.

**Pull request closed without merging**